pelican/settings.py

```diff
@@ -681,3 +681,15 @@ def coerce_overrides(overrides):
                              'load as json', k, v)
                 coerced[k] = json.loads(v)
     return coerced
+
+
+def stringify_plugins(settings):
```

Comment:
I think it is better to put this in `pelican.plugins._utils`. And, I'd prefer this returned a list of strings instead of replacing the setting in-place.